apps/design/backend/src/app.ts

```diff
@@ -240,6 +240,40 @@ function buildApi({ auth, workspace, translator }: AppContext) {
       return ok(election.id);
     },
 
+    async cloneElection(
+      input: WithUserInfo<{
+        srcId: ElectionId;
+        destId: ElectionId;
+        destOrgId: string;
+      }>
+    ): Promise<Result<ElectionId, Error>> {
+      const { election, ballotTemplateId, orgId, precincts, systemSettings } =
+        await store.getElection(input.srcId);
+
+      if (!auth.hasAccess(input.user, orgId)) {
+        throw new grout.GroutError('Access denied', {
+          cause: 'Cannot clone election: invalid source organization.',
+        });
+      }
+      if (!auth.hasAccess(input.user, input.destOrgId)) {
+        throw new grout.GroutError('Access denied', {
```

Comment:
A little odd that this function both returns a `Result` and `throw`s. Why not do this here?

```ts
return err(new grout.GroutError('Access denied', {
  cause: 'Cannot clone election: invalid destination organization.',
}));
```

Reply:
Ah, thanks for catching - no reason; just copied from `createElection` without double-checking.
Are you tied to going the `Result` route? I'm thinking just change that to a normal return to simplify, since this edge case is likely a bad actor or broken client anyway